Makefile

```diff
@@ -7,6 +8,14 @@ AWS_PROFILE := default
 
 PYTEST_OPTS := ''
 
+# Prepare for python version checks
+PYTHON_MIN_VERSION := 3.8
```

Comment:
Let's remove this from this PR.

We removed it because it essentially became just another place to change when making upgrades. Happy to discuss it further if we want to add it back, but let's do that outside of this PR.

Reply:
removed in next commit